exporter/redis.go

```diff
@@ -334,6 +340,36 @@ func parseDBKeyspaceString(db string, stats string) (keysTotal float64, keysExpi
 	return
 }
 
+/*
+	slave0:ip=10.254.11.1,port=6379,state=online,offset=1751844676,lag=0
+	slave1:ip=10.254.11.2,port=6379,state=online,offset=1751844222,lag=0
+*/
+func parseConnectedSlaveString(slaveName string, slaveInfo string) (offset float64, ip string, state string, ok bool) {
+	ok = false
+	if matched, _ := regexp.MatchString(`^slave\d+`, slaveName); !matched {
```

Comment:
Use `strings.HasPrefix()` instead?

Reply:
That was my first approach, but unfortunately, there are other parameters with this prefix.
For example
```
root@test-redis-node2-0:/data# redis-cli info | grep "^slave"
slave_repl_offset:79766288846
slave_priority:100
slave_read_only:1
```

Reply:
ahh, ok, thanks, that makes sense.